mobile/mobile_5f.asm

```diff
@@ -358,11 +358,11 @@ Mobile_CopyDefaultMail:
 
 Mobile_CopyDefaultMailAuthor:
 	ld a, "@"
-	ld de, wc668
+	ld de, wMobileMonMailAuthor
 	ld bc, 5
 	call ByteFill
 	ld hl, Mobile5F_PlayersName
-	ld de, wc668
+	ld de, wMobileMonMailAuthor
 	ld bc, 5
```

Comment:
I'm assuming all of these are the usual 5-long JP names, but if they're not and the 5 is just a coincidence, this should be its own constant.
```suggestion
	ld bc, NAME_LENGTH_JAPANESE - 1
```